vars/ensureImageStream.groovy

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/groovy
+
+def call(Object ctx, String name) {
+  echo "Ensuring image stream ${this._buildName} exists"
```

Comment:
should use the name argument (same on line 9 below)

Reply:
eek, sloppy